docs/docs/_clisettings.mdx

```diff
@@ -13,5 +13,6 @@ Setting name|Definition|Default value
 `output`|Defines the default output when issuing a command. Allowed values are `json`, `text`, `csv`, `md`, `none`|`json`
 `printErrorsAsPlainText`|When output mode is set to `json`, print error messages as plain-text rather than JSON|`true`
 `prompt`|Prompts for missing values in required options and enables interactive selection when multiple values are available for a command that requires a specific value to be retrieved.|`true`
+`promptListPageSize`|By default, lists of choices longer than 7 will be paginated. Use this option to control how many choices will appear on the screen at once.|7
```

Comment:
@martinlingstuyl why not default to the length of the suggestions in the autocomplete? So what we had previously.
As I recall adding this config is a kind of a way to be preped ready to support some consols that don't work well when the prompt output is to long (as @waldekmastykarz mentioned). In this case the user may set a setting to narrow down the options and move on 👍

Reply:
Hi @Adam-it, it now chooses the smallest number, which is either the length of the autocomplete or the cap-size.